pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -271,14 +269,13 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		updateCleanupPolicy = kubectl.RenameRollingUpdateCleanupPolicy
 	}
 	config := &kubectl.RollingUpdaterConfig{
-		Out:            out,
-		OldRc:          oldRc,
-		NewRc:          newRc,
-		UpdatePeriod:   period,
-		Interval:       interval,
-		Timeout:        timeout,
-		CleanupPolicy:  updateCleanupPolicy,
-		UpdateAcceptor: kubectl.DefaultUpdateAcceptor,
+		Out:           out,
+		OldRc:         oldRc,
+		NewRc:         newRc,
+		UpdatePeriod:  period,
```

Comment:
>How about setting maxSurge=1 and maxUnavailability=0 here?

If we set those fields here, then the default behavior from the CLI would be percentage based. As is, the default will be the same behavior as today (absolute up/down by 1). I was trying to preserve the existing defaults.